packages/affiliates/liquidity-mining/yusdeth-weekly-payouts/Week_35_Mining_Rewards.json

```diff
@@ -0,0 +1,138 @@
+{
+  "week": 35,
+  "fromBlock": 12091493,
+  "toBlock": 12136988,
+  "poolAddress": ["0x5e065d534d1daaf9e6222afa1d09e7dac6cbd0f7", "0xf08241998394D61dC0bbbAC767B0B8242549761F"],
+  "blocksPerSnapshot": 256,
+  "umaPerWeek": 5531,
+  "shareHolderPayout": {
```

Comment:
this `shareHolderPayout` contains the sum of the roll between the two contracts.👇